app/assets/stylesheets/pygments.scss

```diff
@@ -63,3 +63,33 @@
 .highlight .ld, /* Literal.Date */
 .highlight .m   /* Literal.Number */
 { color: #ca1655; }
+
+.highlight {
+  position: relative;
+
+  &:focus, &:hover {
+    .highlight__button {
+      visibility: visible;
+    }
+  }
+}
+
+.highlight__button {
+  visibility: hidden;
+  position: absolute;
+  right: .8em;
+  top: 1em;
+  padding: 5px;
+  cursor: pointer;
+
+  > svg {
```

Comment:
Maybe better to stick with class selector to avoid nesting? e.g. `.highlight__copy-icon` or `highlight__button-copy-icon`?

Reply:
SVGs are slightly awkward apply classes so I've kept this as is.